shell/platform/fuchsia/flutter/default_session_connection.cc

```diff
@@ -272,6 +313,15 @@ void DefaultSessionConnection::AwaitVsync(FireCallbackCallback callback) {
   TRACE_DURATION("flutter", "DefaultSessionConnection::AwaitVsync");
   fire_callback_ = callback;
 
+  // Flutter is requesting a vsync here, so mark it as such.
```

Comment:
Let's add inspect for AwaitVsyncForSecondaryCallback as well. It's a new feature that deserves our scrutiny.

Reply:
Done